src/Console/Command.php

```diff
@@ -249,4 +249,35 @@ public function abort($code = self::CODE_ERROR)
     {
         throw new StopException('Command aborted', $code);
     }
+
+    /**
+     * Execute another command with the provided set of arguments.
+     *
+     * @param string|\Cake\Console\Command $command The command class name or command instance.
+     * @param array $args The arguments to invoke the command with.
+     * @param \Cake\Console\ConsoleIo $io The ConsoleIo instance to use for the executed command.
+     * @return int|null The exit code or null for success of the command.
+     */
+    public function executeCommand($command, array $args = [], ?ConsoleIo $io = null)
```

Comment:
`?ConsoleIo` typehint won't work in PHP < 7.1 :stuck_out_tongue:.

Reply:
Dropping the ? should be fine here :)

Reply:
Derp. I forgot that this was for 3.x